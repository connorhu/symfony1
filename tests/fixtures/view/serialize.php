<?php

require_once __DIR__.'/common.inc';

$context = sfContext::getInstance();
$dispatcher = $context->getEventDispatcher();

$parameterHolder = new sfViewParameterHolder(new sfEventDispatcher(), array('foo' => 'bar'));
/** @var sfViewParameterHolder $unserialized */
$unserialized = unserialize(serialize($parameterHolder));
echo implode(',', array_keys($parameterHolder->toArray()));
echo "\n";
echo implode(',', array_keys($unserialized->toArray()));
echo "\n";
echo $parameterHolder->toArray()['foo'];
echo "\n";
echo $unserialized->toArray()['foo'];
echo "\n";
echo $parameterHolder->toArray()['sf_data']['foo'];
echo "\n";
echo $unserialized->toArray()['sf_data']['foo'];
