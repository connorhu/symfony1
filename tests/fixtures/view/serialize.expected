foo,sf_data
foo,sf_data
bar
bar
bar
bar